### ISSUE TITLE
[Feature Request] Support on ZMK

### ISSUE THREAD

**Issue by harshitgoel96** [manna-harbour/qmk_firmware#20]
Can this layout be made available on ZMK?

## Feature Request Type

- [ ] Core functionality
- [ ] Add-on hardware support (eg. audio, RGB, OLED screen, etc.)
- [ ] Alteration (enhancement/optimization) of existing feature(s)
- [x] New behavior

**Comment by manna-harbour** [comment]
Yes! Miryoku ZMK is in development. It currently builds for all ergo splits in ZMK. The keymap can be used now but needs some other work before public release. 

If anyone would like to use it now without redistributing it, just let me know.

**Comment by harshitgoel96** [comment]
> Yes! Miryoku ZMK is in development. It currently builds for all ergo splits in ZMK. The keymap can be used now but needs some other work before public release.
> 
> If anyone would like to use it now without redistributing it, just let me know.

Thanks for the response. In how much time would it be ready for release? Is there any eta at the moment?

I am interested in checking it out.

**Comment by manna-harbour** [comment]
It's waiting on some docs and also a general miryoku reorganisation.  If you have a ZMK board already I can give you access to the repo now. If you happen to be in the Zen GB and are preparing for that I'll make sure it's released before the Zen ships.

**Comment by harshitgoel96** [comment]
I am already using zmk on Lily58 with bluemicro.
Would it be ok if I manually make a keymap myself on zmk ?

**Comment by manna-harbour** [comment]
Yes of course, and there are a few other implementations already. There are also a few people with access to my development version which can be used now. Let's move to discord if you'd like to arrange access.
